matrixbot/matrix.py

```diff
@@ -26,13 +26,15 @@ def __init__(self, settings):
 
         self.settings = settings
         self.period = settings["DEFAULT"]["period"]
-        self.uri = settings["matrix"]["uri"]
-        self.username = settings["matrix"]["username"].lower()
-        self.password = settings["matrix"]["password"]
-        self.room_ids = settings["matrix"]["rooms"]
-        self.domain = self.settings["matrix"]["domain"]
-        self.only_local_domain = self.settings["matrix"]["only_local_domain"]
-        self.super_users = settings["matrix"]["super_users"]
+
+        matrix = settings["matrix"]
+        self.uri = matrix["uri"]
+        self.username = matrix["username"].lower()
+        self.password = matrix["password"]
+        self.room_ids = matrix["rooms"]
+        self.domain = matrix["domain"]
+        self.only_local_domain = matrix["only_local_domain"]
+        self.super_users = "super_users" in matrix and matrix["super_users"]
```

Comment:
I don't understand this change. AFAIK this should be something like: `self.super_users = matrix.get("super_users", [])`

Reply:
What it does is to check "super_users" exists before accessing `matrix["super_users"]`, in case it doesn't exist returns null. In Python the C ternary operator (`a ? a : b`) is written as `a and a or b`. I didn't know about `get`. I like how is written with `get` more.